### ISSUE TITLE
btcd will implement the Compact Block message, BIP152?

### ISSUE THREAD

**Issue by qshuai** [btcsuite/btcd#1208]

**Comment by Roasbeef** [comment]
Yep! We plan to, but haven't got around to it yet due to lack of resources amongst the main contributors. 

If you'd like to get the change started, I'd be happy to review it!

**Comment by qshuai** [comment]
I will attempt to in my spare time.

**Comment by Roasbeef** [comment]
Nice!

**Comment by zquestz** [comment]
Any progress on this? This is also something I am thinking about helping with.

**Comment by Roasbeef** [comment]
No one has picked this up yet, I'd be happy to guide you through an
implementation!

**Comment by jdowning100** [comment]
Any update on this?
